docs/nodes/consensus-full-node.md

```diff
@@ -104,12 +96,43 @@ This will delete all data folders so we can start fresh:
 celestia-appd tendermint unsafe-reset-all --home $HOME/.celestia-app
 ```
 
-### Optional: quick-sync with snapshot
+### Syncing
+
+By default, a consensus node will sync using block sync; that is request, validate
```

Comment:
[question] It looks like block sync is called fast sync on Comet BFT v0.34. Should we use the term "fast sync" and link to https://docs.cometbft.com/v0.34/core/fast-sync here until we upgrade celestia-core to Comet BFT v0.37 which renames fast sync => block sync?

Reply:
I'll defer to @cmwaters on this, and plan to approve this PR after we resolve this 🚢

Reply:
sgtm, lgtm, thanks again